x/proof/keeper/msg_server_submit_proof_test.go

```diff
@@ -350,6 +373,8 @@ func TestMsgServer_SubmitProof_Error_OutsideOfWindow(t *testing.T) {
 }
 
 func TestMsgServer_SubmitProof_Error(t *testing.T) {
+	var claimWindowOpenBlockHash, proofCommitBlockHash []byte
```

Comment:
Are these supposed to receive values somewhere?

Reply:
No, they just need to be declared, it's fine if they're empty.